src/ape/api/accounts.py

```diff
@@ -10,10 +10,10 @@
 
 from .address import AddressAPI
 from .base import abstractdataclass, abstractmethod
-from .contracts import ContractContainer, ContractInstance
 from .providers import ReceiptAPI, TransactionAPI, TransactionType
 
 if TYPE_CHECKING:
+    from ape.contracts import ContractContainer, ContractInstance
```

Comment:
I think you may be able to put this in the first `ape` section, right above line 6

Reply:
Ah nevermind!